### PR TITLE
Update minhphan.json

### PR DESCRIPTION
**Pull request by MinhPhan1203**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.  

## Website Link/Preview

#Sr all ADMIN

### CODE REVIEW

File: domains/minhphan.json [hunk 1]
```diff
@@ -4,6 +4,9 @@
     "email": "phan48788@gmail.com"
   },
   "record": {
-    "URL": "https://minhphan.tech"
+    "A": [
+            "75.2.60.5"
+        ],
+    "TXT" : "dh=78c7c89ac00d07ec34dd5e1d68289a3a877353dd"
```
Comment (Stef-00012) [inline domains/minhphan.json:10]: discord verification TXT goes in a separate domain `_discord.minhphan.is-a.dev`

**Review by MaskDuck — CHANGES_REQUESTED** [review]
we need a preview

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/minhphan.json (hunk 1 above)
Comment (MinhPhan1203) [reply domains/minhphan.json:10]: So i need change TXT to _discord.minhphan.is-a.dev right ?

Comment (MinhPhan1203) [reply domains/minhphan.json:10]: Oh sr i read the docs again sr

**Pull request closed without merging**